source/recipe-individual/scripts/individualCustom.js

```diff
@@ -0,0 +1,301 @@
+class IndividualCustom extends HTMLElement {
+    constructor() {
+        super();
+        let shadow = this.attachShadow({mode: 'open'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8445170)
```suggestion
        let shadow = this.attachShadow({mode: "open"});
```